packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -201,6 +207,23 @@ class Tooltip extends StatefulWidget {
   /// pointer exits the widget.
   final Duration? showDuration;
 
+  /// The [TooltipTriggerMode] that will show the tooltip.
+  ///
+  /// If this property is null, then [TooltipThemeData.triggerMode] is used.
+  /// If [TooltipThemeData.triggerMode] is also null, the default mode is
+  /// [DefaultTooltipTriggerMode] logical pixels on all sides.
```

Comment:
> logical pixels on all sides

This looks like a typo.

Reply:
```suggestion
  /// [TooltipTriggerMode.longPress].
```

It does not look like we need `DefaultTooltipTriggerMode`.

Reply:
why not?
The reason to use it is not because it's used in multiple places, if that is what you mean;

The reason is so that the user may have access to the default value rather than have his own const based on const, and have to worry about the flutter team possibly changing it in the future making his constant deprecated.

imo that should be the behavior for all default values.